### PR TITLE
using a LocateFunctionContextProvider to provide and update the function to the components using it

### PR DESCRIPTION
**Pull request by mfbieber**
Also renamed the folder `Ressources` to `Resources`.

We should identify if we pass any more things through props around that could belong into the `LocateFunctionContext`.

I can already identify, that we could use a `SidebarContentContext`!

### CODE REVIEW

File: src/main/app/src/Components/Landscape/Dashboard/StatusBar.tsx [hunk 1]
```diff
@@ -43,15 +45,13 @@ const StatusBar: React.FC<Props> = ({ setSidebarContent, locateItem, landscape,
...
+    locateFunctionContext.locateFunction(item.fullyQualifiedIdentifier);
+    setSidebarContent(<Item useItem={item} />);
   };
 
   const onGroupClick = (group: IGroup) => {
     findGroup(group.fullyQualifiedIdentifier);
-    setSidebarContent(
-      <Group group={group} assessments={assessments} locateItem={locateItem} locateGroup={findGroup} />
-    );
+    setSidebarContent(<Group group={group} assessments={assessments} />);
```
Comment (bonndan) [inline src/main/app/src/Components/Landscape/Dashboard/StatusBar.tsx:54]: 👍

File: src/main/app/src/Context/LocateFunctionContext.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+
+const initialLocateFunction: (fqi: string) => void = () => {};
+
+export interface LocateFunctionContextType {
+  locateFunction: (fqi: string) => void;
+  setLocateFunction: (locateFunction: () => void) => void;
+}
+export const LocateFunctionContext = React.createContext<LocateFunctionContextType>({
```
[the remark this answers is not in the document]
Comment (mfbieber) [reply src/main/app/src/Context/LocateFunctionContext.tsx:9]: Well, I don't know exactly what I should comment. I think this is not done by just a comment: https://reactjs.org/docs/context.html

I could explain through a video call?

[the remark this answers is not in the document]
Comment (bonndan) [reply src/main/app/src/Context/LocateFunctionContext.tsx:9]: Maybe add the link 😄

**Comment by mfbieber** [comment]
> 👍 yes, the components code looks a lot cleaner without all the copied props

Yes. Besides being less complex now, components are not updated all the time due to props changes. I think what we did before could be considerd as "props drilling". Although this is mostly viewed as an anti-pattern and negatively, I think it is not wrong to pass props around from parents to children. In our case though, it clearly got too complex.

**Review by bonndan — APPROVED** [review]

**Pull request merged**